upgrade/upgrade-scripts/common.sh

```diff
@@ -38,8 +39,13 @@ function get_image_version() {
 	basename "$(get_image_path)"
 }
 
+function get_platform() {
+	dpkg-query -Wf '${Package}' 'delphix-entire-*' | \
```

Comment:
That's a great point. Maybe we can look at `delphix-platform-*` instead of `delphix-entire-*`. Would there ever be a case when that isn't installed? I can't think of any off the top of my head.

Reply:
We could do that, however I'm no sure we should support upgrade at all when `delphix-entire` is missing. We may end up adding some logic to do additional checks, so I think for now just failing when delphix-entire is missing should be fine.

Reply:
I think I'd prefer to leave that policy to the higher level logic, and just have this little helper function be reliable.